templates/default-layout.lucius

```diff
@@ -60,7 +60,7 @@ body {
 
 .col ul,
 .col ol {
-  padding-left: 25px;
+  padding-left: 45px;
```

Comment:
I checked, and type classes etc. looked okay to me. I'm happy to revert this though.